NRasterizer.CLI/Program.cs

```diff
@@ -116,11 +117,15 @@ public void Draw(string rasterizerName, FileInfo fontPath, FileInfo target, stri
 
         public static void Main(string[] args)
         {
-            var rasterizerName = args[0];
-            var fontPath = new FileInfo(args[1]);
-            var target = new FileInfo(args[2]);
-            var text = args[3];
+            //var rasterizerName = args[0];
+            //var fontPath = new FileInfo(args[1]);
+            //var target = new FileInfo(args[2]);
+            //var text = args[3];
 
+            string rasterizerName = "gdi+";
+            FileInfo fontPath = new FileInfo(@"..\..\..\Fonts\CompositeMS.ttf");
+            FileInfo target = new FileInfo(@"test_composite.png");
+            string text = "C";
```

Comment:
Instead of hardcoding string & font, just add line in `build/generate_samples.sh` like so

```bash
mono $PROGRAM nrasterizer Fonts/CompositeMS.ttf samples/C.png C
```

Remeber to also push the ttf file to the `Fonts` directory.